debian/install

```diff
@@ -1,2 +1,3 @@
-debian/homeserver.yaml etc/matrix-synapse
-debian/log.yaml etc/matrix-synapse
+../homeserver.yaml etc/matrix-synapse
+../log.yaml etc/matrix-synapse
+../matrix-synapse.service lib/systemd/system
```

Comment:
rather than do this, can we give the `--name=matrix-synapse` argument to `dh_installinit`, which will also mean that we can preserve the name of the `.default` file?

Reply:
(i'm also not sure that /lib is the right place for this file, but that becomes moot if we do it with installinit instead)

Reply:
`/lib` is the right spot, it's symlinked into `/etc` by `systemctl enable`.

Reply:
ok the docs on https://manpages.debian.org/stretch/debhelper/dh_installinit.1.en.html convince me that /lib is the right spot, but not that we should be doing it this way.

Reply:
fwiw we can do it the "right way" when we rename the package, I didn't find a way for dh_systemd to install it under a name that wasn't the package name. :/ Once it's just `matrix-synapse` again, we'll be able to use dh_systemd and it'll suck less.